charts/fpl-case-service/values.preview.template.yaml

```diff
@@ -7,9 +7,10 @@ java:
   ingressHost: ${SERVICE_FQDN}
   livenessDelay: 50
   environment:
+    IDAM_API_URL: http://api-idam-fpl.service.core-compute-preview.internal
     CORE_CASE_DATA_API_URL: http://${SERVICE_NAME}-data-store-api
     IDAM_CLIENT_ID: fpl_case_service
-    IDAM_CLIENT_REDIRECT_URI: https://fpl-case-service-*.service.core-compute-aat.internal/oauth2/callback
+    IDAM_CLIENT_REDIRECT_URI: https://localhost:9000/oauth2/callback
```

Comment:
Hmm why this is localhost?
Is it overriden? maybe we shouldn't put the value here then? I find it confusing.

Reply:
That redirect URL doesn't matter as redirect never happens. It is just to match whitelisted URL on IDAM side. It matches https://github.com/hmcts/cnp-flux-config/blob/master/k8s/preview/family-public-law/idam.yaml#L36

Reply:
Yeah this bit is something I don't understand in those authentication flows, why the parameters which doesn't matter are required to be passed, yeah the redirect neve happens as this is not browser login, should there be another endpoint? 🤔 need to read more about it to understand it properly!